packages/core/src/sass/add-or-manage.scss

```diff
@@ -36,10 +36,11 @@
         }
         &__deposit-warn {
             width: 70%;
-            margin: 2.8rem auto 0;
+            margin: 2.8rem auto 8rem;
```

Comment:
Does this work for both responsive and desktop?

Reply:
@mikita-deriv  ?

Reply:
Yes, it works

Reply:
test